assets/questions/rle.js

```diff
@@ -78,7 +85,10 @@ def DecodeEncodedString(word):
     performanceTests: []
   }, {
     instructions: [
-      'You need to make sure that your code handles short strings correctly.'
+      [
+        'Next, double-check your code to make sure it handles short strings. ',
+        'Ideally, these strings should also be as small as possible.',
```

Comment:
I think the also is redundant, there -- it implies a separate condition that's not needed.